### PR TITLE
Add user defined types to graphql-schema endpoint

### PR DESCRIPTION
**Pull request by henneberger**

### CODE REVIEW

File: graphql/keyspace.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ var dataCenterInput = graphql.NewInputObject(graphql.InputObjectConfig{
 })
 
 var keyspaceType = graphql.NewObject(graphql.ObjectConfig{
-	Name: "Keyspace",
+	Name: "keyspaceName",
```
Comment (mpenick) [inline graphql/keyspace.go:49]: Is this change intentional?

File: graphql/keyspace.go [hunk 2]
```diff
@@ -112,7 +112,7 @@ func (sg *SchemaGenerator) buildKeyspaceQuery(singleKeyspace string) *graphql.Ob
 	return graphql.NewObject(graphql.ObjectConfig{
 		Name: "Query",
 		Fields: graphql.Fields{
-			"keyspace": &graphql.Field{
+			"keyspaceName": &graphql.Field{
```
Comment (mpenick) [inline graphql/keyspace.go:115]: Same here. Is this intentional?

Comment (henneberger) [reply graphql/keyspace.go]: Ah no. Fixed.
